fireworks_poe_bot/fw_poe_image_bot.py

```diff
@@ -98,73 +102,74 @@ def _log_info(self, payload: Dict):
     async def get_response(
         self, query: QueryRequest
     ) -> AsyncIterable[Union[PartialResponse, ServerSentEvent]]:
-        if len(query.query) == 0:
-            yield ErrorResponse(allow_retry=False, text="Empty query")
-            return
+        orig_api_key = self.client.api_key
```

Comment:
is this needed for anything?

Reply:
Yes, there's an unmodified use in the "finally" block below